spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -122,7 +122,7 @@ def get_parser():
 class AnalyzeLesion:
     def __init__(self, fname_mask, fname_sc, fname_ref, path_template, path_ofolder, perslice, verbose):
         self.fname_mask = fname_mask
-        self.interpolated_midsagittal_slice = None  # target float sagittal slice number used for the interpolation. This number is based on the spinal cord center of mass.
+        self.interpolated_midsagittal_slice = None  # target float sagittal slice number used for the interpolation in the RPI orientation
```

Comment:
Maybe now would be a good time to change `self.interpolated_midsagittal_slice` to be `self.interpolated_midsagittal_slice_RPI`? 

I think in a previous PR you had suggested this change actually, but I think I suggested not to do this because it was a big PR and I wanted to keep the diff small. :sweat_smile: 

But I think it might be appropriate in this PR, since it will help distinguish between the RPI slice (used for metric calculations) and the orig orientation slice (used for QC/output).